homeassistant/components/habitica/services.yaml

```diff
@@ -15,3 +15,26 @@ api_call:
       example: '{"text": "Use API from Home Assistant", "type": "todo"}'
       selector:
         object:
+cast_skill:
+  fields:
+    config_entry:
+      required: true
+      selector:
+        config_entry:
+          integration: habitica
+    skill:
+      required: true
+
```

Comment:
```suggestion
```

Reply:
yamllint wants a new line at the end of file